docs/first-line-support-developer-runbook.md

```diff
@@ -0,0 +1,75 @@
+# First-line support developer runbook
+
```

Comment:
TODO: Add Verify key rotation once https://github.com/DFE-Digital/claim-additional-payments-for-teaching/pull/998 is merged.